.vscode/launch.json

```diff
@@ -18,34 +19,21 @@
 			}
 		},
 		{
+			"name": "Run Extension Tests",
+			"type": "extensionHost",
 			"request": "launch",
-			"name": "Launch CLI",
-			"type": "pwa-node",
-			"program": "${workspaceFolder}/index.ts",
-			"cwd": "${workspaceFolder}", // If you want to run taq from a different location, set the address in this line
-			"runtimeExecutable": "deno",
-			"runtimeArgs": [
-				"run",
-				"--inspect-brk",
-				"--allow-run",
-				"--allow-write",
-				"--allow-read",
-				"--allow-env",
-				"--allow-net",
-				"--import-map",
-				"${workspaceFolder}/import_map.json"
-			],
+			"runtimeExecutable": "${execPath}",
```

Comment:
I'm not sure what `${execPath}` refers to here. What does this mean and what does it do?

Reply:
Hey @alexzbusko, more details about it here - https://code.visualstudio.com/docs/editor/variables-reference
${execPath} - the path to the running VS Code executable